repobee_sanitizer/_sanitize_repo.py

```diff
@@ -31,22 +31,20 @@ class SanitizeRepo(plug.Plugin):
     def _sanitize_repo(
         self, args: argparse.Namespace, api: None,
     ) -> Optional[Mapping[str, List[plug.Result]]]:
-        if not args.file_list.is_file():
-            raise plug.PlugError(f"No such file: {args.file_list}")
+
+        if args.file_list:
+            if not args.file_list.is_file():
+                raise plug.PlugError(f"No such file: {args.file_list}")
+            file_relpaths = _parse_file_list(args.file_list)
+        elif args.discover_files:
+            file_relpaths = _discover_dirty_files(args.repo_root)
```

Comment:
Nested if statements but this seems like the best way to do it? 

Also: Even though it SHOULD not be possible right now, should we add an else statement in case none of those happen?

Reply:
I did not know that assert operator exsited. Thats super neat!